### DISCUSSION TITLE
Adding Exult to F-Droid (Android)

### DISCUSSION THREAD

**Discussion by msmigiel** [exult/exult#611] [General]
When searching F-Droid, I do not see this app available. It would be great if it could be included in F-Droid so updating it could be much easier. It is recommended to ask the author first - please let me know your thoughts. If we agree, I can raise an issue here: https://gitlab.com/fdroid/rfp/-/issues

I think F-Droid might be easier, but having this in the Google Play store ultimately would be really nice too. With this, updates would be easier to find out about and apply.

**Comment by DominusExult** [comment]
Please, go ahead. You will probably need to figure out if you want releases only or the pre-release snapshots :)

**Reply by DominusExult** [reply]
I've converted it into a discussion which is a better fit, I guess
